### PR TITLE
feat(frontend): style header menu

### PR DESCRIPTION
**Pull request by Elweyn**
## 🍰 Pullrequest
Screenshot 2024-04-11 at 14 52 18

### Issues
- None

### Todo

- [X] Change favicon
- [X] Add logo to header
- [X] Style log out button in header
- [x] Style get-room button in main content

### CODE REVIEW

**Review by mahula — CHANGES_REQUESTED** [review]
Please add a headline welcomming the user to the Weltencafé.

File: frontend/src/components/buttons/TextButtonInput.vue
Comment (ulfgebhardt) [inline frontend/src/components/buttons/TextButtonInput.vue:1]: is this used?

File: frontend/src/components/menu/LogoAvatar.vue
Comment (ulfgebhardt) [inline frontend/src/components/menu/LogoAvatar.vue:1]: is this used or is `LogoImage.vue` used?

File: frontend/src/components/buttons/TextButtonInput.vue
Comment (Elweyn) [reply frontend/src/components/buttons/TextButtonInput.vue:1]: Removed it, you right it is not

File: frontend/src/components/menu/LogoAvatar.vue
Comment (Elweyn) [reply frontend/src/components/menu/LogoAvatar.vue:1]: Removed it, you right it is not

**Review by ulfgebhardt — APPROVED** [review]

**Pull request merged**